crates/sncast/src/main.rs

```diff
@@ -335,9 +334,7 @@ async fn run_async_command(
                 Ok(())
             }
             account::Commands::Delete(delete) => {
-                config.account = delete
-                    .name
-                    .ok_or_else(|| anyhow!("required argument --name not provided"))?;
+                config.account = delete.name;
```

Comment:
We can pass `delete.name` to `starknet_commands::account::delete::delete` below so this line is not necessary